content/en/tracing/serverless_functions/_index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Trace Serverless Functions
+kind: documentation
+description: 'Send traces from your Serverless Functions to Datadog'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
+
+In hybrid architectures, Serverless functions can form an important core of requests your application processes. Including these functions in your Distributed Traces can be critical for detecting performance bottlenecks and service outages in your distributed systems.
+
+## Install the AWS Integration
```

Comment:
We should change this section to be like "Choose your tracing library" since that's what this is really about

Reply:
or something more generall like "Serverless Disributed Tracing"